src/mon/MgrMap.h

```diff
@@ -179,6 +179,29 @@ class MgrMap
     return false;
   }
 
+  bool module_enabled(const std::string &module_name) const
+  {
+    for (const auto &name : modules) {
```

Comment:
Since `modules` is a `std::set`, you can make this function a one-liner `modules.find(module_name) != modules.end()`

Reply:
@jcsp fixed.